pkg/types/urlsmap_test.go

```diff
@@ -113,3 +112,45 @@ func TestNewURLsMapIPV6(t *testing.T) {
 		t.Errorf("cluster = %#v, want %#v", c, wc)
 	}
 }
+
+func TestNewURLsMapFromStringMap1(t *testing.T) {
+	mss := make(map[string]string)
+	urlsMap, err := NewURLsMapFromStringMap(mss, ",")
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+	s := ""
+	um, err := NewURLsMap(s)
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+
+	if um.String() != urlsMap.String() {
+		t.Errorf("Expected:\n%+v\ngot:\n%+v", um, urlsMap)
+	}
+}
+
+func TestNewURLsMapFromStringMap2(t *testing.T) {
```

Comment:
Okay, I've pushed a new version. Anything else that is not good feel free
to add a subsequent commit on top.

Cheers,
James

On Sat, May 14, 2016 at 10:41 AM, Xiang Li <notifications@github.com> wrote:

> In pkg/types/urlsmap_test.go
> <https://github.com/coreos/etcd/pull/5298#discussion_r63277087>:
>
> > +	urlsMap, err := NewURLsMapFromStringMap(mss, ",")
> > +	if err != nil {
> > +		t.Errorf("Unexpected error: %v", err)
> > +	}
> > +	s := ""
> > +	um, err := NewURLsMap(s)
> > +	if err != nil {
> > +		t.Errorf("Unexpected error: %v", err)
> > +	}
> > +
> > +	if um.String() != urlsMap.String() {
> > +		t.Errorf("Expected:\n%+v\ngot:\n%+v", um, urlsMap)
> > +	}
> > +}
> > +
> > +func TestNewURLsMapFromStringMap2(t *testing.T) {
>
> TestNewURLsMapFromStringMap
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/etcd/pull/5298/files/ebdb2b25c2586d062270c9a868bad8a0af2853d6#r63277087>
>